CalendarApp/NUnitTestProject/BookingServiceTests.UpdateTest.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using NUnit.Framework;
+using System.Collections.Generic;
+using Google.Apis.Calendar.v3.Data;
+using BookingRoom.Models.GoogleCalendar;
+using Google.Apis.Calendar.v3;
+using System.Linq;
+using BookingRoom.Helpers;
+
+namespace NUnitTestProject
+{
+    [TestFixture]
+    public partial class BookingServiceTests : BaseIntegrationTest
+    {
+        [Test]
+        public void BookingService_UpdateEvent_UpdatesEventCorrectly()
+        {
+            var calendarId = AppSettingsHelper.GetAppSetting(AppSetingsConst.TestCalendar);
+
+            var beforeUpdateEventList = _calendarService.Events.List(calendarId).Execute().Items;
+            if (beforeUpdateEventList == null)
+                throw new Exception("Event list are empty!");
+            var testEvent = beforeUpdateEventList[0];
+
+            var currentSummary = testEvent.Summary; //remember current state
+            var currentDesctiption = testEvent.Description;
+            testEvent.Summary = "ChangedSummary"; //chenage event
+            testEvent.Description = "ChengedDescription";
```

Comment:
I'd suggest to update existing value. Like testEvent.Description = $"Chenged{currentDesctiption}";